lib/app_web/live/app_live.ex

```diff
@@ -108,15 +116,246 @@ defmodule AppWeb.AppLive do
     })
 
     AppWeb.Endpoint.broadcast(@topic, "update", :update)
-    {:noreply, assign(socket, editing: nil)}
+    {:noreply, assign(socket, editing: nil, editing_timers: [])}
+  end
+
+  @impl true
+  def handle_event(
+        "update-item-timer",
+        %{
+          "timer_id" => id,
+          "index" => index,
+          "timer_start" => timer_start,
+          "timer_stop" => timer_stop
+        },
+        socket
+      )
+      when timer_stop == "" do
+    timer_changeset_list = socket.assigns.editing_timers
+    index = String.to_integer(index)
+    changeset_obj = Enum.at(timer_changeset_list, index)
+
+    try do
```

Comment:
This is quite a lot of business logic to include in `app_live.ex` ...
Is there a reason we can't put most of the `timer` related code in `/lib/app/timer.ex` ?

Reply:
You're right, it should go into the `timer.ex` file, as it pertains to it. I was going to refactor the code and hopefully make the flow easier to understand before pushing to review 😃